src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/client/activitiesappointments/ActivitiesAppointmentsClientTest.kt

```diff
@@ -71,21 +74,6 @@ class ActivitiesAppointmentsClientTest {
     appointments.map { it.internalLocation?.id } containsExactlyInAnyOrder setOf(2000, 3000)
   }
 
-  @Test
-  fun `should get no prisoner appointments at locations when not video link locations`() {
```

Comment:
A valid test still? BVLS should only see VIDE-type locations, even for VLPMs?

Reply:
No, the API now returns all appointment types.  We then filter them out in due course in the code where used.